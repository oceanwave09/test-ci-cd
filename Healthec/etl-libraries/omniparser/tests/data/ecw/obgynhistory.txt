apuid|RecordID|PatientId|PatientAccountNo|encounterId|EncounterDate|ProviderNPI|Ecwstructid|Question|QuestionID|Answers|AnswerID|ItemName|ItemID|CategoryName|CategoryID|MainCategoryName|MainCategoryID
280807|280807.17502.673750.1448|17502|17502|673750|2022-10-24|1235205972|0|symptoms:|1448|irritability|3686|Premenstrual Syndrome|269102|GYN History|138212|GYNHistory|
280807|280807.17502.623962.1445|17502|17502|623962|2022-10-24|1235205972|0|Lasts:|1445|||Menstruation:|269101|GYN History|138212|GYNHistory|
280807|280807.17502.714249.1442|17502|17502|714249|2022-10-17|1669957486|0|Age of onset of maternal menarche:|1442|||Menarche:|269100|GYN History|138212|GYNHistory|
280807|280807.17502.714249.3761|17502|17502|714249|2022-10-17|1669957486|0|Received a Clinical Breast Exam|3761|Yes||Breast Health Screening|275371|GYN History|138212|GYNHistory|
280807|280807.17502.2364838.1161|17502|17502|2364838|2022-10-25|1306337332|0|when done?|1161|1 year ago|1990|Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2364838.1162|17502|17502|2364838|2022-10-25|1306337332|0|Encouraged patient to follow up annually|1162|Yes||Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2364838.1163|17502|17502|2364838|2022-10-25|1306337332|0|Medical Release form given|1163|No||Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2364838.1698|17502|17502|2364838|2022-10-25|1306337332|0|Date:|1698|09/19/2022||LMP:|256236|GYN History|138212|GYNHistory|
280807|280807.17502.2364838.4299|17502|17502|2364838|2022-10-25|1306337332|0|Age (years):|4299|12||Menarche (onset):|412967|GYN History|138212|GYNHistory|
280807|280807.17502.2362781.1161|17502|17502|2362781|2022-10-25|1982168415|0|when done?|1161|3 years ago|1992|Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2362781.1698|17502|17502|2362781|2022-10-25|1982168415|0|Date:|1698|10/13/2022||LMP:|256236|GYN History|138212|GYNHistory|
280807|280807.17502.2362781.4299|17502|17502|2362781|2022-10-25|1982168415|0|Age (years):|4299|||Menarche (onset):|412967|GYN History|138212|GYNHistory|
280807|280807.17502.2369193.1699|17502|17502|2369193|2022-10-27|1285299289|0|total:|1699|1||Pregnancies|278295|Ob History|138211|ObHistory|
280807|280807.17502.2369193.1700|17502|17502|2369193|2022-10-27|1285299289|0|children:|1700|1||Total living|138214|Ob History|138211|ObHistory|
280807|280807.17502.2369193.1701|17502|17502|2369193|2022-10-27|1285299289|0|how many:|1701|1||Full-term delivery|199644|Ob History|138211|ObHistory|
280807|280807.17502.2370397.1161|17502|17502|2370397|2022-10-26|1659865087|0|when done?|1161|2 years ago|1991|Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2370397.1697|17502|17502|2370397|2022-10-26|1659865087|0|patient not in age range:|1697|not medically indicated|2897|Mammogram:|256248|GYN History|138212|GYNHistory|
280807|280807.17502.2370397.1698|17502|17502|2370397|2022-10-26|1659865087|0|Date:|1698|10/08/2022||LMP:|256236|GYN History|138212|GYNHistory|
280807|280807.17502.2370397.4299|17502|17502|2370397|2022-10-26|1659865087|0|Age (years):|4299|||Menarche (onset):|412967|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.1161|17502|17502|2363066|2022-10-17|1871036319|0|when done?|1161|> 3 years ago|14243|Last PAP:|256239|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.1405|17502|17502|2363066|2022-10-17|1871036319|0|history:|1405|none|2504|Sexual Abuse:|279741|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.1698|17502|17502|2363066|2022-10-17|1871036319|0|Date:|1698|07/14/2022||LMP:|256236|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.1699|17502|17502|2363066|2022-10-17|1871036319|0|total:|1699|4||Pregnancies|278295|Ob History|138211|ObHistory|
280807|280807.17502.2363066.1700|17502|17502|2363066|2022-10-17|1871036319|0|children:|1700|4||Total living|138214|Ob History|138211|ObHistory|
280807|280807.17502.2363066.1701|17502|17502|2363066|2022-10-17|1871036319|0|how many:|1701|4||Full-term delivery|199644|Ob History|138211|ObHistory|
280807|280807.17502.2363066.4299|17502|17502|2363066|2022-10-17|1871036319|0|Age (years):|4299|12||Menarche (onset):|412967|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.4653|17502|17502|2363066|2022-10-17|1871036319|0|Breast Self-Exam|4653|UTD|14244|Breast Health Screening:|278067|GYN History|138212|GYNHistory|
280807|280807.17502.2363066.970|17502|17502|2363066|2022-10-17|1871036319|0|Last mammogram:|970|||Mammogram:|256248|GYN History|138212|GYNHistory|
280807|280807.17502.2349480.1161|17502|17502|2349480|2022-10-18|1548638877|0|when done?|1161|> 3 years ago|14243|Last PAP:|256239|GYN History|138212|GYNHistory|
