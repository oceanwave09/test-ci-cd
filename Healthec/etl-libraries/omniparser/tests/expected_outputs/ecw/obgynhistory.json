[
	{
		"answers": "irritability",
		"answers_id": "3686",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-24",
		"encounter_id": "673750",
		"item_id": "269102",
		"item_name": "Premenstrual Syndrome",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1008dfbee72b30f54c1badcc355a8395",
		"question": "symptoms:",
		"question_id": "1448",
		"record_id": "280807.17502.673750.1448",
		"row_id": "ff2dd66d-6caf-4b52-a3c0-920abd1457ea"
	},
	{
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-24",
		"encounter_id": "623962",
		"item_id": "269101",
		"item_name": "Menstruation:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "1008dfbee72b30f54c1badcc355a8395",
		"question": "Lasts:",
		"question_id": "1445",
		"record_id": "280807.17502.623962.1445",
		"row_id": "310803e9-7e25-4c22-be77-59dc45c97611"
	},
	{
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "714249",
		"item_id": "269100",
		"item_name": "Menarche:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "30367468ccf7c74816fc7ee65bc6fb94",
		"question": "Age of onset of maternal menarche:",
		"question_id": "1442",
		"record_id": "280807.17502.714249.1442",
		"row_id": "4873628b-98ab-4e66-9fd4-fcaa8fdefe66"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "714249",
		"item_id": "275371",
		"item_name": "Breast Health Screening",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "30367468ccf7c74816fc7ee65bc6fb94",
		"question": "Received a Clinical Breast Exam",
		"question_id": "3761",
		"record_id": "280807.17502.714249.3761",
		"row_id": "39967ec3-e5a0-46a6-ab3b-3a24dc3b99fe"
	},
	{
		"answers": "1 year ago",
		"answers_id": "1990",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2364838",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "82678c1a276fa93a372f13e78370334a",
		"question": "when done?",
		"question_id": "1161",
		"record_id": "280807.17502.2364838.1161",
		"row_id": "31e547b5-6cb6-41a9-9d43-f46b32f2ae4a"
	},
	{
		"answers": "Yes",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2364838",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "82678c1a276fa93a372f13e78370334a",
		"question": "Encouraged patient to follow up annually",
		"question_id": "1162",
		"record_id": "280807.17502.2364838.1162",
		"row_id": "0ef7ca0e-da88-443e-a98f-a6c16a461afc"
	},
	{
		"answers": "No",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2364838",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "82678c1a276fa93a372f13e78370334a",
		"question": "Medical Release form given",
		"question_id": "1163",
		"record_id": "280807.17502.2364838.1163",
		"row_id": "327af787-7d1b-44be-babf-182a30166723"
	},
	{
		"answers": "09/19/2022",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2364838",
		"item_id": "256236",
		"item_name": "LMP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "82678c1a276fa93a372f13e78370334a",
		"question": "Date:",
		"question_id": "1698",
		"record_id": "280807.17502.2364838.1698",
		"row_id": "cfa76b3e-eb95-4c3a-944c-8260d51b548f"
	},
	{
		"answers": "12",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2364838",
		"item_id": "412967",
		"item_name": "Menarche (onset):",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "82678c1a276fa93a372f13e78370334a",
		"question": "Age (years):",
		"question_id": "4299",
		"record_id": "280807.17502.2364838.4299",
		"row_id": "d5b8059f-5160-48e3-9187-388cc4745280"
	},
	{
		"answers": "3 years ago",
		"answers_id": "1992",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2362781",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7fa50feb361c7b79058dacee5db9187d",
		"question": "when done?",
		"question_id": "1161",
		"record_id": "280807.17502.2362781.1161",
		"row_id": "08069033-0036-4065-b5b2-5a4af5ae0ecb"
	},
	{
		"answers": "10/13/2022",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2362781",
		"item_id": "256236",
		"item_name": "LMP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7fa50feb361c7b79058dacee5db9187d",
		"question": "Date:",
		"question_id": "1698",
		"record_id": "280807.17502.2362781.1698",
		"row_id": "1d3abf91-68b2-4e1c-8194-651c363e6b8c"
	},
	{
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-25",
		"encounter_id": "2362781",
		"item_id": "412967",
		"item_name": "Menarche (onset):",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "7fa50feb361c7b79058dacee5db9187d",
		"question": "Age (years):",
		"question_id": "4299",
		"record_id": "280807.17502.2362781.4299",
		"row_id": "99e8499d-a7e1-4263-b81b-31ba6348acb3"
	},
	{
		"answers": "1",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-27",
		"encounter_id": "2369193",
		"item_id": "278295",
		"item_name": "Pregnancies",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "910040e36e7b04008bec7800df5416ac",
		"question": "total:",
		"question_id": "1699",
		"record_id": "280807.17502.2369193.1699",
		"row_id": "ee7dd945-1f0c-439b-98ea-8eefdc1a3b3d"
	},
	{
		"answers": "1",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-27",
		"encounter_id": "2369193",
		"item_id": "138214",
		"item_name": "Total living",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "910040e36e7b04008bec7800df5416ac",
		"question": "children:",
		"question_id": "1700",
		"record_id": "280807.17502.2369193.1700",
		"row_id": "9d8ffb84-f821-4b00-b417-8489ae837cdb"
	},
	{
		"answers": "1",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-27",
		"encounter_id": "2369193",
		"item_id": "199644",
		"item_name": "Full-term delivery",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "910040e36e7b04008bec7800df5416ac",
		"question": "how many:",
		"question_id": "1701",
		"record_id": "280807.17502.2369193.1701",
		"row_id": "a7fc3fd0-f3a2-47f6-b611-1ab117c18349"
	},
	{
		"answers": "2 years ago",
		"answers_id": "1991",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-26",
		"encounter_id": "2370397",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cbebe235ad5bd5eaf17945939f11a026",
		"question": "when done?",
		"question_id": "1161",
		"record_id": "280807.17502.2370397.1161",
		"row_id": "fdde84d0-6c93-4830-a5b0-41220d6ec3f1"
	},
	{
		"answers": "not medically indicated",
		"answers_id": "2897",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-26",
		"encounter_id": "2370397",
		"item_id": "256248",
		"item_name": "Mammogram:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cbebe235ad5bd5eaf17945939f11a026",
		"question": "patient not in age range:",
		"question_id": "1697",
		"record_id": "280807.17502.2370397.1697",
		"row_id": "685f009f-1310-4aff-957b-5a9154e12f2a"
	},
	{
		"answers": "10/08/2022",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-26",
		"encounter_id": "2370397",
		"item_id": "256236",
		"item_name": "LMP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cbebe235ad5bd5eaf17945939f11a026",
		"question": "Date:",
		"question_id": "1698",
		"record_id": "280807.17502.2370397.1698",
		"row_id": "adbd02f2-16b4-4fe0-b4cc-790d25e10a60"
	},
	{
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-26",
		"encounter_id": "2370397",
		"item_id": "412967",
		"item_name": "Menarche (onset):",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "cbebe235ad5bd5eaf17945939f11a026",
		"question": "Age (years):",
		"question_id": "4299",
		"record_id": "280807.17502.2370397.4299",
		"row_id": "56281ce9-4f0a-43e6-aa4e-2e6e830e22bb"
	},
	{
		"answers": "> 3 years ago",
		"answers_id": "14243",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "when done?",
		"question_id": "1161",
		"record_id": "280807.17502.2363066.1161",
		"row_id": "4228fd4d-193c-4787-9717-a3fcfe928672"
	},
	{
		"answers": "none",
		"answers_id": "2504",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "279741",
		"item_name": "Sexual Abuse:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "history:",
		"question_id": "1405",
		"record_id": "280807.17502.2363066.1405",
		"row_id": "250142bd-c7ea-41ca-8eae-6fa4a358fea1"
	},
	{
		"answers": "07/14/2022",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "256236",
		"item_name": "LMP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "Date:",
		"question_id": "1698",
		"record_id": "280807.17502.2363066.1698",
		"row_id": "20da5efe-62fa-4fd2-bb1d-7f5c04667a51"
	},
	{
		"answers": "4",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "278295",
		"item_name": "Pregnancies",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "total:",
		"question_id": "1699",
		"record_id": "280807.17502.2363066.1699",
		"row_id": "8b231d6e-eacd-42cb-95b2-6d8cf0a868e2"
	},
	{
		"answers": "4",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "138214",
		"item_name": "Total living",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "children:",
		"question_id": "1700",
		"record_id": "280807.17502.2363066.1700",
		"row_id": "9573b9d7-3339-4d4a-bd37-70f5bb177e4a"
	},
	{
		"answers": "4",
		"apuid": "280807",
		"category_id": "138211",
		"category_name": "Ob History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "199644",
		"item_name": "Full-term delivery",
		"main_category_name": "ObHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "how many:",
		"question_id": "1701",
		"record_id": "280807.17502.2363066.1701",
		"row_id": "6f1558dc-3f38-4d2d-8f57-7307da79cead"
	},
	{
		"answers": "12",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "412967",
		"item_name": "Menarche (onset):",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "Age (years):",
		"question_id": "4299",
		"record_id": "280807.17502.2363066.4299",
		"row_id": "f0833868-f310-43eb-9496-634c101c141a"
	},
	{
		"answers": "UTD",
		"answers_id": "14244",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "278067",
		"item_name": "Breast Health Screening:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "Breast Self-Exam",
		"question_id": "4653",
		"record_id": "280807.17502.2363066.4653",
		"row_id": "43f17f49-d812-4747-8fee-2df598da1020"
	},
	{
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-17",
		"encounter_id": "2363066",
		"item_id": "256248",
		"item_name": "Mammogram:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "0562f47637286d5b3574316f87bfcb7a",
		"question": "Last mammogram:",
		"question_id": "970",
		"record_id": "280807.17502.2363066.970",
		"row_id": "aa055d1a-2b75-4bf7-a8e8-def61dbb9f64"
	},
	{
		"answers": "> 3 years ago",
		"answers_id": "14243",
		"apuid": "280807",
		"category_id": "138212",
		"category_name": "GYN History",
		"ecw_struct_id": "0",
		"encounter_date": "2022-10-18",
		"encounter_id": "2349480",
		"item_id": "256239",
		"item_name": "Last PAP:",
		"main_category_name": "GYNHistory",
		"patient_account_no": "17502",
		"patient_id": "17502",
		"provider_npi": "dd09c68b99d94d2066925d644b365a8a",
		"question": "when done?",
		"question_id": "1161",
		"record_id": "280807.17502.2349480.1161",
		"row_id": "5823da6e-be28-4e05-b21a-6bc5353a7f8f"
	}
]
